File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/codex-sl/max_sublist_sum.java
--- 
+++ 
@@ -3,7 +3,7 @@
     int max_so_far = 0;
 
     for (int x : arr) {
-        max_ending_here = max_ending_here + x;
+        max_ending_here = Math.max(0, max_ending_here + x);
         max_so_far = Math.max(max_so_far, max_ending_here);
     }
 

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/incoder-1/max_sublist_sum.py
--- 
+++ 
@@ -3,7 +3,7 @@
     max_so_far = 0
 
     for x in arr:
-        max_ending_here = max_ending_here + x
+        max_ending_here = max(max_ending_here+x, 0)
         max_so_far = max(max_so_far, max_ending_here)
 
     return max_so_far
